repos/rust-lang/bors.toml

```diff
@@ -12,8 +12,3 @@ mods = "maintain"
 pattern = "main"
 ci-checks = ["Test", "Test Docker"]
 required-approvals = 0
-
-[[branch-protections]]
```

Comment:
We have the `allowed-merge-teams` on branches which give teams the ability to push against protected branches. I think it's fine to merge this for now, but perhaps a better solution is adding a `allowed-merge-individuals` option and adding yourself to that? That way we can more easily document this work around.

Reply:
Allowed merge teams doesn't allow force pushes, IIRC, the way sync-team is configured. It only allows to select a subset of teams that have write+ perms on the repo, which will have push/merge rights to the branch.